ui/src/app/pages/access/data-access-requirements.tsx

```diff
@@ -604,32 +609,32 @@ export const DataDetail = (props: { icon: string; text: string }) => {
 export const DataAccessRequirements = fp.flow(withProfileErrorModal)(
   (spinnerProps: WithSpinnerOverlayProps) => {
     // Local State
-    // At any given time, at most two modules will be clickable during initial registration:
-    //  1. The active module, which we visually direct the user to with a CTA
-    //  2. The next required module, which may diverge when the active module is optional.
+    // At any given time, at most two modules will be active during initial registration:
+    //  1. The focused module, which we visually direct the user to with a CTA
+    //  2. The next required module, which may diverge when the focused module is optional.
     // This configuration allows the user to skip the optional CT section.
-    const [activeModule, setActiveModule] = useState(null);
-    const [clickableModules, setClickableModules] = useState([]);
+    const [focusedModule, setFocusedModule] = useState(null);
+    const [activeModules, setActiveModules] = useState([]);
 
     // Local Variables
     const { profile, reload } = useStore(profileStore);
     const {
       config: { unsafeAllowSelfBypass },
     } = useStore(serverConfigStore);
 
-    const query = useQuery();
-    const code = query.get('code');
-    const token = query.get('token');
+    const urlParams = new URLSearchParams(window.location.search);
```

Comment:
Changed this to not use `useQuery`, because on line 373 (where query parameters are being stripped), this was removing query parameters from the URL but not from react-router history. It seems like this works better in the newer version of React-router (v6) by utilizing [useNavigate](https://reactrouter.com/en/main/hooks/use-navigate).

When the query parameters were not being removed, the rasCallback functionality would be called multiple times and result in an error.

If anyone is aware of a better solution, please let me know.